integration-cli/docker_cli_events_test.go

```diff
@@ -401,7 +460,7 @@ func (s *DockerSuite) TestEventsFilterContainer(c *check.C) {
 
 	checkEvents := func(id string, events []string) error {
 		if len(events) != 4 { // create, attach, start, die
-			return fmt.Errorf("expected 3 events, got %v", events)
+			return fmt.Errorf("expected 4 events, got %v", events)
```

Comment:
Alternatively I could change it to use assert which would handle the messaging without this issue.